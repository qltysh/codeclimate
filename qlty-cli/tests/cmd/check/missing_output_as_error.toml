bin.name = "qlty"
args = ["check", "--all", "--no-cache"]
status.code = 3
env.remove = ["RUST_BACKTRACE"]