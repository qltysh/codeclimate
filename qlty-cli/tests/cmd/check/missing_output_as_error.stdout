Lint error exists: Exited with code 0 .qlty/out/invocations/invoke-[..].yaml
[..]The plugin crashed for some reason[..]

Checked 1 file
